utils/_features.py

```diff
@@ -2266,3 +2266,13 @@ def container_guardrail(test_object):
         """
         pytest.mark.features(feature_id=309)(test_object)
         return test_object
+
+    @staticmethod
+    def iast_sink_untrusted_deserialization(test_object):
+        """
+        IAST Sink: Untrusted Deserialization
+
+        https://feature-parity.us1.prod.dog/#/?feature=311
+        """
+        pytest.mark.features(feature_id=311)(test_object)
```

Comment:
We need to add it to feature parity, this id is provisional

Reply:
Updated to the new feature_id